services/brig/src/Brig/Team/Template.hs

```diff
@@ -62,23 +61,24 @@ loadTeamTemplates o = readLocalesDir defLocale (templateDir gOptions) "team" $ \
                 <*> readTemplate fp "email/invitation.txt"
                 <*> readTemplate fp "email/invitation.html"
                 <*> pure (emailSender gOptions)
-                <*> readText (fp <> "/email/sender.txt"))
+                <*> readText fp "email/sender.txt")
         <*> (CreatorWelcomeEmailTemplate (tCreatorWelcomeUrl tOptions)
                 <$> readTemplate fp "email/new-creator-welcome-subject.txt"
                 <*> readTemplate fp "email/new-creator-welcome.txt"
                 <*> readTemplate fp "email/new-creator-welcome.html"
                 <*> pure (emailSender gOptions)
-                <*> readText (fp <> "/email/sender.txt"))
+                <*> readText fp "email/sender.txt")
         <*> (MemberWelcomeEmailTemplate (tMemberWelcomeUrl tOptions)
                 <$> readTemplate fp "email/new-member-welcome-subject.txt"
                 <*> readTemplate fp "email/new-member-welcome.txt"
                 <*> readTemplate fp "email/new-member-welcome.html"
                 <*> pure (emailSender gOptions)
-                <*> readText (fp <> "/email/sender.txt"))
+                <*> readText fp "email/sender.txt")
   where
     gOptions = general (emailSMS o)
     tOptions = team (emailSMS o)
     tUrl     = template $ tInvitationUrl tOptions
 
     defLocale = setDefaultLocale (optSettings o)
     readTemplate = readTemplateWithDefault (templateDir gOptions) defLocale "team"
+    readText = readTextWithDefault (templateDir gOptions) defLocale "team"
```

Comment:
Looks like `readText` is defined on two scope levels?  Do we allow that?

Reply:
sorry, scratch that.  `readTextWithDefault` used to be called `readText`, but it's not any more.